app/controllers/items_controller.rb

```diff
@@ -15,6 +10,11 @@ def create
     end
   end
 
+  def index
+    @item = Item.new
+    @category = Category.new
```

Comment:
Indeed. Setting more than 1 instance variable in a controller action is indicative of bad architecture.

Take at look at Sandi Metz's [Rules](https://robots.thoughtbot.com/sandi-metz-rules-for-developers)